web/src/components/Spacer.tsx

```diff
@@ -1,7 +1,8 @@
 import styled from '@emotion/styled'
+import { Divider } from '@mui/material'
 
-const Spacer = styled.hr<{ borderColor: string }>`
+const Spacer = styled(Divider)`
```

Comment:
🔧 I think export does not warrant its own file anymore, I think we should just apply the margin separately if needed and delete this file.

Reply:
This makes sense ... Do you wanna call it `WithDivider` ?

Reply:
> Do you wanna call it WithDivider ?

Those are unrelated things. Most of the usages of `Spacer` you should be able to just replace with `Divider`. In some cases you might need to apply a margin.